internal/proto/reader.go

```diff
@@ -171,6 +185,29 @@ func (r *Reader) ReadString() (string, error) {
 	}
 }
 
+func (r *Reader) readStringReplyToWriter(line []byte, output io.Writer) (int, error) {
```

Comment:
Is the logic in this function cloned/copied from elsewhere?

Reply:
Yes, from `readStringReplyBuffered` below.
The main idea here is to have adjustable buffer to work without fear of not enough size in buffered method